assets/src/apps/delivery/layouts/deck/DeckLayoutFooter.tsx

```diff
@@ -431,7 +431,7 @@ const DeckLayoutFooter: React.FC = () => {
               </button>
             </div>
             <style type="text/css" aria-hidden="true" />
-            <div className="content">
+            <div className="content" style={{display:'inline-table'}}>
```

Comment:
im not sure this is always going to be good? I think in spr it applies the width and height perhaps